BLMM/BLMM.py

```diff
@@ -155,15 +85,11 @@ def _main(argv=None):
         # Append to list 
         futures.append(future_b)
 
-    # Completed jobs
-    completed = as_completed(futures)
+    # wait for results 
+        wait(futures)
```

Comment:
Since we don't need a return, we can use the `wait` function instead of as_completed, don't you think?

Reply:
Before writing the code how I had with the return, I tried to do it this way. However, I found I was experiencing some strange issues. 

When I looked into it I found that for some reason, without the return, dask wasn't running all of the futures. I then tried calling results on these futures and found that in some instances dask was running them twice. Is there any way you could check here that all of the futures are being run, and are being run only once? I had a lot of trouble with this which is how I landed on the version in the old code.

Reply:
fair enough let me undo this change